cse/templates/hpa.yaml

```
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```

```suggestion
apiVersion: {{ ge .Capabilities.KubeVersion.Minor "23" | ternary "autoscaling/v2" "autoscaling/v2beta2" }}
```

This should set it to v2 if they're on 1.23+, @jfuen10 would you suggest doing this a different way potentially?